packages/wasm/crate/tests/test_mock_db.rs

```diff
@@ -0,0 +1,72 @@
+use futures::executor::block_on;
+use penumbra_keys::keys::AddressIndex;
+use wasm_bindgen_test::wasm_bindgen_test;
+
+use penumbra_wasm::database::interface::Database;
+use penumbra_wasm::database::mock::MockDb;
+
+wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
+
+#[wasm_bindgen_test]
+fn test_get_and_put() {
+    let db = MockDb::new();
+    let table_name = "test_table";
+
+    let key = "test_key";
+    let value = AddressIndex::new(1);
+
+    block_on(db.put_with_key(table_name, key, &value)).unwrap();
```

Comment:
how does `block_on` work?

Reply:
I didn't occur to me that I could just define the test fn as async and use await syntax. This was an unnecessary workaround. Will update.